Library/Formula/zorba.rb

```diff
@@ -0,0 +1,35 @@
+class Zorba < Formula
+  homepage "http://www.zorba.io/"
+  url "https://github.com/28msec/zorba/archive/3.0.tar.gz"
+  sha1 "c444cde689600aab1172b4974348dd6626e92261"
+
+  option "with-big-integer", "Use 64 bit precision instead of arbitrary precision for performance"
+  option "with-ssl-verification", "Enable SSL peer certificate verification"
+
+  depends_on "cmake" => :build
+  depends_on "swig" => :build
+  depends_on "flex"
+  depends_on "icu4c"
+  depends_on "xerces-c"
+
+  needs :cxx11
+
+  def install
+    ENV.cxx11
+    ENV["CMAKE_PREFIX_PATH"] = "#{Formula["icu4c"].prefix}:#{Formula["flex"].prefix}"
+
+    cmake_args = std_cmake_args
+    cmake_args << "-DZORBA_VERIFY_PEER_SSL_CERTIFICATE=ON" if build.with? "ssl-verification"
+    cmake_args << "-DZORBA_WITH_BIG_INTEGER=ON" if build.with? "big-integer"
```

Comment:
The rationale given in [the documentation](https://github.com/28msec/zorba/blob/master/doc/zorba/build_parameters.dox#L39-L45) for the first one is *"This is because not all systems have a complete or valid set of root certificates against which to validate said server certificates"*. I'm not able to judge if that's a good reason or not, so I didn't alter the default.

For the second one, there is a performance vs accuracy tradeof, and I think it's best to stick with the default so you get the same results by default as you'd get if you install it from source yourself (on a linux box, say).